src/framework/components/camera/component.js

```diff
@@ -107,6 +107,9 @@ import { PostEffectQueue } from './post-effect-queue.js';
  * camera should belong. Don't push/pop/splice or modify this array, if you want to
  * change it, set a new one instead. Defaults to [LAYERID_WORLD, LAYERID_DEPTH,
  * LAYERID_SKYBOX, LAYERID_UI, LAYERID_IMMEDIATE].
+ * @property {number} stopPostprocessingLayer Layer ID of layer which stops postprocessing of the camera.

```

Comment:
I'm not keen on `stopPostprocessingLayer ` as a name. Trying to think of something better...

Reply:
It's not my favorite name, but the best I could come up. Alternatives anybody?

Reply:
Maybe a flag to say whether it's included, with UI layer defaulted as not included? Not sure if that would cause other problems or if we would have to add warnings if the user sets the layer order such that an included layer was after the UI layer?

Reply:
ideally I want to specify which layer is last for postprocessing, not just a bool to say to stop on UI
example: you can postprocess environment, but not characters and UI.

Reply:
* `stopPostLayer`
* `stopPostFxLayer`
* `stopPostEffectsLayer`
* `stopPostprocessLayer`
* `stopPostprocessingLayer`
* `disablePostLayer`
* `disablePostFxLayer`
* `disablePostEffectsLayer`
* `disablePostprocessLayer`
* `disablePostprocessingLayer`

@slimbuck Any opinions?

Reply:
`postProcessStopLayer` would probably make most sense to me, but I guess we want the verb to go first?

Reply:
maybe this:
disablePostEffectsLayer

the post effect base class is PostEffect, so that matches

Reply:
Agreed!

Reply:
and done.